net-im/signal-desktop-bin/signal-desktop-bin-1.40.0.ebuild

```diff
@@ -0,0 +1,114 @@
+	dev-libs/atk
+	dev-libs/expat
+	dev-libs/glib
+	dev-libs/nspr
+	dev-libs/nss
+	media-libs/alsa-lib
+	media-libs/mesa[X(+)]
+	net-print/cups
+	sys-apps/dbus[X]
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+:3[X]
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libXcomposite
+	x11-libs/libXcursor
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXi
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	x11-libs/libXScrnSaver
+	x11-libs/libXtst
+	x11-libs/pango
+	sound? (
+		|| (
+			media-sound/pulseaudio
+			media-sound/apulse
+		)
+	)
+"
+
+QA_PREBUILT="opt/Signal/signal-desktop
+	opt/Signal/chrome-sandbox
+	opt/Signal/crashpad_handler
+	opt/Signal/libffmpeg.so
+	opt/Signal/libGLESv2.so
+	opt/Signal/libnode.so
+	opt/Signal/libVkICD_mock_icd.so
+	opt/Signal/libvk_swiftshader.so
+	opt/Signal/libvulkan.so
+	opt/Signal/swiftshader/libGLESv2.so
+	opt/Signal/resources/app.asar.unpacked/node_modules/curve25519-n/build/Release/curve.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/libsignal-client/build/libsignal_client_linux.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/@journeyapps/sqlcipher/lib/binding/napi-v6-linux-x64/node_sqlite3.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/zkgroup/node_modules/ref-napi/build/Release/binding.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/ref-napi/build/Release/binding.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/ringrtc/build/linux/libringrtc.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/ffi-napi/build/Release/ffi_bindings.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/sharp/build/Release/sharp.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/sharp/vendor/8.10.5/lib/libvips-cpp.so.42
+	opt/Signal/resources/app.asar.unpacked/node_modules/sharp/vendor/8.10.5/lib/libvips.so.42
+	opt/Signal/resources/app.asar.unpacked/node_modules/zkgroup/libzkgroup.so"
+
+RESTRICT="splitdebug"
+
+S="${WORKDIR}"
+
+src_prepare() {
+	default
+	sed -e 's| --no-sandbox||g' \
+		-i usr/share/applications/signal-desktop.desktop || die
+	unpack usr/share/doc/signal-desktop/changelog.gz
+}
+
+src_install() {
+	insinto /
+	dodoc changelog
+	doins -r opt
+	insinto /usr/share
+
+	if has_version media-sound/apulse[-sdk] && ! has_version media-sound/pulseaudio; then
+		sed -i 's/Exec=/Exec=apulse /g' usr/share/applications/signal-desktop.desktop
+	fi
+
+	doins -r usr/share/applications
+	doins -r usr/share/icons
+	fperms +x /opt/Signal/signal-desktop /opt/Signal/chrome-sandbox
+	fperms u+s /opt/Signal/chrome-sandbox
+	pax-mark m opt/Signal/signal-desktop opt/Signal/chrome-sandbox
+
+	dosym ../../opt/Signal/${MY_PN} /usr/bin/${MY_PN}
+	dosym ../../usr/lib64/libEGL.so opt/Signal/libEGL.so
+	dosym ../../../usr/lib64/libEGL.so opt/Signal/swiftshader/libEGL.so
+}
+
+pkg_postinst() {
+	xdg_pkg_postinst
+	elog "For using the tray icon on compatible desktop environments, start Signal with '--start-in-tray' or '--use-tray-icon'."
```

Comment:
Let's multi-line it, 
https://devmanual.gentoo.org/ebuild-writing/messages/index.html

> In all cases, assume that the user's terminal is no wider than 79 columns, and that the elog,einfo, ewarn and eerror functions will occupy 4 columns with their fancy leading markers.